packages/repository-json-schema/src/filter-json-schema.ts

```diff
@@ -34,7 +34,8 @@ export function getFilterJsonSchemaFor(modelCtor: typeof Model): JsonSchema {
 
       limit: {
         type: 'integer',
-        minimum: 0,
+        minimum: 1,
+        examples: [100],
```

Comment:
Nice. Can you please add examples for `offset` and `skip` properties too?

Reply:
We could specify `0` as example, but this is assumed by the explorer by default. Other values would break the user experience of the explorer as it sets this values as default, which might not be useful.

Reply:
Makes sense 👍